pom.xml

```diff
@@ -54,7 +54,7 @@
   </scm>
 
   <properties>
-    <nexus.version>2.9.1-02</nexus.version>
+    <nexus.version>2.14.2-SNAPSHOT</nexus.version>
```

Comment:
no - the fix went in after the release... I guess that should be 2.14.3-SNAPSHOT - fixing now